source/equipment/magic-items/potion-of-healing.rst

```diff
@@ -7,8 +7,21 @@ Potion of Healing
 
 *Potion, rarity varies*
 
-You regain hit points when you drink this potion. The number of hit
-points depends on the potion's rarity, as shown in the Potions of
-Healing table. Whatever its potency, the potion's red liquid glimmers
-when agitated.
+You regain hit points when you drink this potion. The
+number of hit points depends on the potion’s rarity, as
+shown in the Potions of Healing table. Whatever its
+potency, the potion’s red liquid glimmers when agitated.
+
+
+Potions of Healing
```

Comment:
@eepMoody @jasonmcalpin should this be done as `..table:: Potions of Healing`? That's seems more semantic and that's how I did it in #126, but we don't seem to have styling for table titles yet. This is not a blocker for merging, we can go back and fix all the table titles later.